scripts/install-deps-debian-sid.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 set -e
-export DEBIAN_FRONTEND=noninteractive
 
+DEBIAN_SID_DEPS=ca-certificates gcc libc6-dev make automake wget git golang-go coreutils cpio squashfs-tools realpath autoconf file xz-utils patch bc locales libacl1-dev libssl-dev
+
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
Even with `-y`, `apt-get` can ask question on the terminal. `DEBIAN_FRONTEND=noninteractive` tells `apt-get` that it must not do that.